src/modules/feed/components/Transaction/LikeButton.js

```diff
@@ -0,0 +1,89 @@
+import { h } from "preact";
+import { Button, Icon } from "semantic-ui-react";
+import gql from "graphql-tag";
+import { Mutation } from "react-apollo";
+
+import settings from "../../../../config/settings";
```

Comment:
For readability, would it help to link to it from the root? (using a webpack resolve alias?)